### PR TITLE
l10n - add Saraiki (skr) to languages.py

### PR DESCRIPTION
**Pull request by bcolsson**
Saraiki locale was added to Firefox starting from 128. This change is to ensure backend languages file matches the frontend changes being added in [this PR](https://github.com/mozilla/addons-frontend/pull/13189).

This adds the following:
Locale: skr
English: Saraiki
Native: سرائیکی

### CODE REVIEW

**Comment by diox** [comment]
Is this about adding a locale to AMO itself, for AMO to be available in that language, or is it about adding a language pack (and/or dictionary) for that locale in https://addons.mozilla.org/en-US/firefox/language-tools/ ?

**Comment by bcolsson** [comment]
> Is this about adding a locale to AMO itself, for AMO to be available in that language, or is it about adding a language pack (and/or dictionary) for that locale in https://addons.mozilla.org/en-US/firefox/language-tools/ ?

This is for adding a language pack for the locale to language tools. From my understanding it's really only necessary to add it to the front end (which is adding in [this PR](https://github.com/mozilla/addons-frontend/pull/13189)), however when we added Santali last time I believe you asked for a matching one for the backend (which is the reason for this PR).

**Comment by bcolsson** [comment]
@diox - Thank's for the feedback and explanation. I've modified the PR to move skr into `ADDITIONAL_PRODUCT_LANGUAGES`.

File: src/olympia/core/languages.py [hunk 1]
```diff
@@ -214,6 +214,7 @@
         'native': '\u1c65\u1c5f\u1c71\u1c5b\u1c5f\u1c72\u1c64',
     },
     'si': {'english': 'Sinhala', 'native': '\u0dc3\u0dd2\u0d82\u0dc4\u0dbd'},
+    'skr': {'english': 'Saraiki', 'native': '\u0633\u0631\u0627\u0626\u06cc\u06a9\u06cc'},
```
Comment (diox) [inline src/olympia/core/languages.py:217]: Linting doesn't like that line, try running `ruff format` on the file or apply

```suggestion
    'skr': {
        'english': 'Saraiki',
        'native': '\u0633\u0631\u0627\u0626\u06cc\u06a9\u06cc',
    },
```

**Comment by diox** [comment]
Thanks!

**Pull request merged**